modules/axhal/src/platform/x86_pc/time.rs

```diff
@@ -53,7 +63,16 @@ pub(super) fn init_early() {
         }
     }
 
-    unsafe { INIT_TICK = core::arch::x86_64::_rdtsc() };
+    let rtc_lock = SpinNoIrq::new(());
+    let _guard = rtc_lock.lock();
```

Comment:
Why use a lock here?

Reply:
Getting real time requires multiple reads from CMOS port, which requires IRQ safe.
If IRQ is not enabled here, it can be removed

Reply:
Since IRQ is disabled during platform initialization, this lock has been removed.